common/build-style/python3-module.sh

```diff
@@ -25,8 +25,8 @@ do_build() {
 
 do_check() {
 	if python3 -c 'import pytest' >/dev/null 2>&1; then
-		${make_check_pre} PYTHONPATH="$(cd build/lib* && pwd)" \
-			python3 -m pytest ${make_check_args} ${make_check_target}
+		PYTHONPATH="$(cd build/lib* && pwd)"
+		${make_check_pre} python3 -m pytest ${make_check_args} ${make_check_target}
```

Comment:
```suggestion
		PYTHONPATH="$(cd build/lib* && pwd)" \
			${make_check_pre} python3 -m pytest ${make_check_args} ${make_check_target}
```
Thanks for noticing this